src/backup_restore.cpp

```diff
@@ -289,15 +300,17 @@ void BackupAndRestore::backupAndRestoreIpzVpd(types::IPZVpdMap& io_srcVpdMap,
             // restore config JSON.
             if (l_dstBinaryValue == l_defaultBinaryValue)
             {
-                // TODO: Write on hardware,
-                // Write on Dbus,
-                // To keep the data in sync between hardware and parsed map
-                // updating the io_dstVpdMap. This should only be done if write
-                // on hardware returns success. utils::writeKeyword(i_dstPath,
-                // types::IpzData(l_dstRecordName, l_dstKeywordName
-                // l_srcBinaryValue));
+                // Update keyword's value on hardware
+                /* To keep the data in sync between hardware and parsed map
+                 updating the io_dstVpdMap. This should only be done if write
+                 on hardware returns success.*/
+                std::shared_ptr<Parser> l_vpdParser =
```

Comment:
auto can be used here as well. Also please use a blank line between blocks of code.

Reply:
done